testData/ontologies/flat.json5

```diff
@@ -181,197 +315,273 @@
             "description": "Position of the magnified area",
             "type": "string",
             "default": "TopHalf",
-            "enum": ["FullScreen", "Lens", "LeftHalf", "RightHalf", "TopHalf", "BottomHalf", "Custom"]
+            "enum": [
+                "FullScreen",
+                "Lens",
+                "LeftHalf",
+                "RightHalf",
+                "TopHalf",
+                "BottomHalf",
+                "Custom"
+            ],
+            "enumLabels": [
+                "Full Screen",
+                "Lens",
+                "Left Half",
+                "Right Half",
+                "Top Half",
+                "Bottom Half",
+                "Custom"
+            ]
         },
-        "http://registry.gpii.net/common/invertColours": {
-            "title": "Invert colours",
-            "description": "Whether to invert colours",
+        "http://registry.gpii.net/common/mouseEmulation/enabled": {
+            "title": "Mouse emulation",
+            "description": "Whether to enable/disable the mouse emulation",
             "type": "boolean",
             "default": false
         },
-        "http://registry.gpii.net/common/showCrosshairs": {
-            "title": "Show crosshairs",
-            "description": "Whether to show crosshairs",
+        "http://registry.gpii.net/common/mouseTrailing": {
+            "title": "Mouse Trailing",
+            "description": "Amount of mouse trailing",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 10
+        },
+        "http://registry.gpii.net/common/nightScreen": {
+            "title": "Night Screen",
+            "description": "Reduce the blue emitted from the screen.",
             "type": "boolean",
             "default": false
         },
-        "http://registry.gpii.net/common/volume": {
-            "title": "Volume",
-            "description": "General volume of the operating system",
+        "http://registry.gpii.net/common/onScreenKeyboard/enabled": {
+            "title": "On-Screen keyboard",
+            "description": "Whether to enable/disable the on-screen keyboard",
+            "type": "boolean",
+            "default": false
+        },
+        "http://registry.gpii.net/common/pitch": {
+            "title": "Pitch rate",
+            "description": "Text to speech output pitch rate",
             "type": "number",
-            "min": 0,
-            "max": 1
+            "default": 0.5,
+            "minimum": 0,
+            "maximum": 1,
+            "multipleOf": 0.1
         },
-        "http://registry.gpii.net/common/language": {
-            "title": "Language",
-            "description": "Language of the system",
-            "type": "string",
-            "default": "en",
+        "http://registry.gpii.net/common/punctuationVerbosity": {
+            "title": "Punctuation verbosity",
+            "description": "Level of puctuacion verbosity when producing text to speech",
```

Comment:
Typo puctuation

Reply:
Shoddy work, a typo in punctuation should have at least had a stray punctuation mark in it for comedic effect.  Anyway,  it's fixed.